bootstrap.sh

```
@@ -1 +1,3 @@
 #!/bin/bash
+
+kubectl apply -f ./infrastructure/statefulset.yml
```

Problems:
- The path used here (`./infrastructure/statefulset.yml`) does not match the repository manifest path/name (`.infrastructure/statefulSet.yml`) so `kubectl apply` will likely fail. Fix the path or apply the directory.
- The script does not create the kind cluster (missing `kind create cluster --config cluster.yml`), does not apply Secrets/ConfigMaps first, and does not deploy the application or its Secret. Update the script to perform all these actions, wait for resources to be ready, and fail fast on errors.
Suggested actions to implement here (examples):
- `kind create cluster --config cluster.yml`
- `kubectl --context kind-yourcluster wait --for=condition=Ready node --all --timeout=120s`
- `kubectl apply -f .infrastructure/`  # ensures namespace, secret, configmap, service, statefulset are applied
- `kubectl rollout status statefulset/mysql -n mysql --timeout=300s`
- apply application namespace/secret/deployment
Add `set -euo pipefail` at the top for safer execution.